packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -76,8 +87,16 @@ class SystemRulesTable extends React.Component {
         };
 
     }
+
     removeRefIdPrefix = (refId) => {
-        return refId.split('xccdf_org.ssgproject.content_profile_')[1];
+        const splitRefId = refId.toLowerCase().split('xccdf_org.ssgproject.content_profile_')[1];
```

Comment:
Again - we have to call `.toLowerCase`. Another problem is that we were not doing the right split here - if splitting fails, we have to return the original refId